app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb

```diff
@@ -0,0 +1,80 @@
+<%= render("waste_carriers_engine/shared/back", back_path: back_copy_cards_payment_forms_path(reg_identifier: @transient_registration.reg_identifier)) %>
+
+<div class="text">
+  <%= form_for(@copy_cards_payment_form) do |f| %>
+    <% if flash[:error].present? %>
+      <div class="error-summary" role="alert">
+        <h2 class="heading-medium error-summary-heading"><%= t(".payment_error_heading") %></h2>
+
+        <ul class="error-summary-list">
+          <p><%= flash[:error] %></p>
+        </ul>
+      </div>
+    <% end %>
+
+    <%= render("waste_carriers_engine/shared/errors", object: @copy_cards_payment_form) %>
+
+    <h1 class="heading-large"><%= t(".heading") %></h1>
+
+    <div class="form-group">
+      <table>
+        <tbody>
+          <tr>
+            <th>
+              <span class="strong">
+                <% if @transient_registration.temp_cards > 1 %>
```

Comment:
Did you know you can do this:

```
t(".total_cost", count: @transient_registration.temp_cards)
```

And then in the YAML have:

```
cost:
  one: "1 card"
  other: "%{count} cards"
```

And it'll handle all the logic for you? It's pretty handy :)

Reply:
😮 I didn't! TIL! Thank :) will make the change!